### DISCUSSION TITLE
Public release

### DISCUSSION THREAD

**Discussion by Mte90** [snicco/fortress#14] [Q&A] [answered]
I discovered today https://snicco.io/blog/introducing-fortress-vaults-and-pillars but you don't have any social accounts so to get updates from your blog is not easy.
I added right now the RSS feed but it is difficult to keep up with what you are doing it.

The article talks about 1000 enthusiasts before september 2023 but I think that didn't happened.
It is possible to release it?

I would like to integrate in wordpress plugin boilerplate https://wpbp.github.io/ that has 731 stars on github and can help on promoting.

**Comment by calvinalkan** [comment, marked as answer]
It's not releaed outside of Fortess yet.

The best way to stay up to date would be Twitter:

https://twitter.com/calvinalkan
